modules/osdk-cli-reference-cleanup.adoc

```diff
@@ -0,0 +1,41 @@
+// Module included in the following assemblies:
+//
+// * operators/operator_sdk/osdk-cli-reference.adoc
+
+[id="osdk-cli-reference-cleanup_{context}"]
+= cleanup
+
+The `operator-sdk cleanup` command destroys and removes resources that were created for an Operator that was deployed with the `run` command.
+
+== packagemanifests
+`cleanup packagemanifests` subcommand destroys an Operator that was deployed with OLM by using the `run packagemanifests` command.
+
+.`packagemanifests` arguments
+[options="header",cols="1,3"]
+|===
+|Arguments |Description
+
+|`--include` (string)
+|The file path to Kubernetes resource manifests, for example, Role, Subscription. These supplement or override defaults generated by run/cleanup.
+
+|`--install-mode` (string)
+|The `OperatorGroup` is created with the specified `InstallMode`. Format: `InstallModeType[=ns1,ns2[, ...]]``
+
+|`--kubeconfig` (string)
+|The file path to a Kubernetes configuration file. Defaults to the location specified by `$KUBECONFIG`, or to default file rules if not set
+
+|`--olm-namespace` (string)
+|The namespace where the OLM is installed. Default: `olm`.
+
+|`--operator-namespace` (string)
+|The namespace where the operator resources are created. The namespace must already exist in the cluster, or be defined in a manifest passed to `--include`.
```

Comment:
Please uppercase Operator in this description, the next, and spot check the others too.

Please also consider "...or be defined in a manifest that is passed..."  Or, maybe "...or be defined in a manifest that is specified with the `--include` argument."  Up to you.